File: ExpenseTrackerWebService/Model/TotalPriceModel.cs
namespace ExpenseTrackerWebService.Model;
public class TotalPriceModel {
    public double total { get; set; }
    public string name { get; set; }

    public TotalPriceModel(double total, string name) {
        this.total = total;
        this.name = name;
    }
}

File: expenses_tracker_flutter/lib/model/total_price_model.dart
import 'package:intl/intl.dart';

class TotalPrice {
  double total;
  String name;

  TotalPrice(this.total, this.name);

  TotalPrice.fromJson(Map<String, dynamic> json)
    : total = json['total'] as double,
      name = json['name'] as String;

  Map<String, dynamic> toJson() => {
    'total' : total,
    'name': name
  };
}